pyproject.toml

```
@@ -1,3 +1,33 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+[project]
+name = "rwth_i6_hlt_pytorch_models"
+version = "0.0.1"
+authors = [
+{ name="Christopy Lüscher", email="luescher@cs.rwth-aachen.de" },
+{ name="Nick Rossenbach", email="rossenbach@cs.rwth-aachen.de" },
+{ name="Benedikt Hilmes", email="hilmes@cs.rwth-aachen.de" },
+{ name="Jingjing Xu", email="jxu@cs.rwth-aachen.de" },
+{ name="Mohammad Zeineldeen", email="zeineldeen@cs.rwth-aachen.de" },
+{ name="Albert Zeyer", email="zeyer@cs.rwth-aachen.de" },
+{ name="Peter Vieting", email="vieting@cs.rwth-aachen.de" },
+{ name="Simon Berger", email="berger@cs.rwth-aachen.de" },
+{ name="Eugen Beck", email="ebeck@apptek.com" },
+{ name="Ping Zheng", email="pzheng@apptek.com" },
+{ name="Wilfried Michel", email="wmichel@apptek.com" },
+]
+description = "A collection of PyTorch NN models and parts. TO be used with the RETURNN framework."
```

```suggestion
description = "A collection of PyTorch NN models and parts. To be used with the RETURNN framework."
```
I would remove the reference to returnn, i think these are generic pytorch model definitions that can be used with any pytorch based tool, right?